src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -892,6 +901,54 @@ bool isObjectInitializerInitializedObjectTarget(IFlowCaptureReferenceOperation r
                 } && left == referenceSyntax;
             }
 
+            bool isInterpolatedStringArgumentCapture(IFlowCaptureReferenceOperation reference)
+            {
+                if (reference.Language != LanguageNames.CSharp)
+                {
+                    return false;
+                }
+
+#pragma warning disable IDE0055 // Fix formatting
+                return reference is
+                       {
+                           Parent: IArgumentOperation
+                           {
+                               Parent: IObjectCreationOperation
+                               {
+                                   Parent: IFlowCaptureOperation,
+                                   Constructor.ContainingType: INamedTypeSymbol ctorContainingType,
+                                   Arguments: { Length: >= 3 } arguments,
+                                   Syntax: CSharp.Syntax.InterpolatedStringExpressionSyntax or CSharp.Syntax.BinaryExpressionSyntax
```

Comment:
>CSharp.Syntax.InterpolatedStringExpressionSyntax [](http://example.com/codeflow?start=43&length=48)

Should we worry about parenthesis and supressions?

Reply:
Probably.